configs/openSUSE/dbus-services.toml

```diff
@@ -1,270 +1,750 @@
-    # powerdevil chargethreshold helper (bsc#1176474)
-    "org.kde.powerdevil.chargethresholdhelper.service",
-    "org.kde.powerdevil.chargethresholdhelper.conf",
-    # plasma5-disks smartmon helper
-    "org.kde.kded.smart.service",
-    "org.kde.kded.smart.conf"
+[[FileDigestGroup]]
+package = "cups"
+type = "dbus"
+note = ""
+bug = "bnc#515977"
+skips = [
+    "/etc/dbus-1/system.d/cups.conf",
+]
+
+[[FileDigestGroup]]
+package = "upower"
+type = "dbus"
+note = "legacy: not audited"
+skips = [
+    "/usr/share/dbus-1/system-services/org.freedesktop.UPower.service",
+    "/usr/share/dbus-1/system.d/org.freedesktop.UPower.conf",
+]
+
+[[FileDigestGroup]]
+package = "packagekit"
+type = "dbus"
+note = "legacy: not audited"
+skips = [
+    "/etc/dbus-1/system.d/org.freedesktop.PackageKit.conf",
+    "/usr/share/dbus-1/system-services/org.freedesktop.PackageKit.service",
+]
+
+[[FileDigestGroup]]
+package = "networkmanager-pptp"
+type = "dbus"
+note = "legacy: not audited"
+skips = [
+    "/etc/dbus-1/system.d/nm-pptp-service.conf",
+]
+
+[[FileDigestGroup]]
+package = "gdm"
+type = "dbus"
+note = "legacy: not audited"
+skips = [
+    "/etc/dbus-1/system.d/gdm.conf",
+]
+
+[[FileDigestGroup]]
+package = "udisks2"
+type = "dbus"
+note = "legacy: not audited"
+bug = "bnc#742751"
+skips = [
+    "/usr/share/dbus-1/system-services/org.freedesktop.UDisks2.service",
+    "/usr/share/dbus-1/system.d/org.freedesktop.UDisks2.conf",
+]
+
+[[FileDigestGroup]]
+package = "scmon"
+type = "dbus"
+note = "legacy: not audited"
+skips = [
+    "/etc/dbus-1/system.d/com.novell.Pkcs11Monitor.conf",
+]
+
+[[FileDigestGroup]]
+package = "systemd"
+type = "dbus"
+note = "legacy: not audited"
+bug = "bnc#641924"
+skips = [
+    "/usr/share/dbus-1/services/org.freedesktop.systemd1.service",
+    "/usr/share/dbus-1/system-services/org.freedesktop.systemd1.service",
+    "/usr/share/dbus-1/system.d/org.freedesktop.systemd1.conf",
+    "/usr/lib/systemd/system/dbus-org.freedesktop.hostname1.service",
+    "/usr/share/dbus-1/system-services/org.freedesktop.hostname1.service",
+    "/usr/share/dbus-1/system.d/org.freedesktop.hostname1.conf",
+    "/usr/share/dbus-1/system.d/org.freedesktop.login1.conf",
+    "/usr/lib/systemd/system/dbus-org.freedesktop.login1.service",
+    "/usr/share/dbus-1/system-services/org.freedesktop.login1.service",
+    "/usr/share/dbus-1/system.d/org.freedesktop.timedate1.conf",
+    "/usr/lib/systemd/system/dbus-org.freedesktop.timedate1.service",
+    "/usr/share/dbus-1/system-services/org.freedesktop.timedate1.service",
+    "/usr/share/dbus-1/system.d/org.freedesktop.locale1.conf",
+    "/usr/lib/systemd/system/dbus-org.freedesktop.locale1.service",
+    "/usr/share/dbus-1/system-services/org.freedesktop.locale1.service",
+]
+
+# gconf2
+#"org.gnome.GConf.Defaults.service",
+#"org.gnome.GConf.Defaults.conf",
+
+# system-config-printer (bnc#694640)
+#"com.redhat.NewPrinterNotification.conf",
+#"com.redhat.PrinterDriversInstaller.conf",
```

Comment:
this is also still existing in `system-config-printer-applet` from `system-config-printer`